dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -1068,4 +1070,64 @@ public final Response delete(@Context final HttpServletRequest  request,
         }
     }
 
+    /**
+     * Copies container to the specified host
+     *
+     * @param request
+     * @param response
+     * @param id       id identifier to copy.
+     * @return
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
+    @POST
+    @Path("/{id}/_copy")
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public ResponseEntityContainerView copy(@Context final HttpServletRequest request,
+            @Context final HttpServletResponse response,
+            @PathParam("id") final String id) throws DotDataException, DotSecurityException {
+
+        final InitDataObject initData = new WebResource.InitBuilder(webResource)
+                .requestAndResponse(request, response).requiredBackendUser(true)
+                .rejectWhenNoUser(true).init();
+        final User user = initData.getUser();
+        final Host host = WebAPILocator.getHostWebAPI().getCurrentHostNoThrow(request);
+        final PageMode pageMode = PageMode.get(request);
+
+        if (!UtilMethods.isSet(id)) {
+
+            Logger.error(this, "The container id is required");
+            throw new IllegalArgumentException("The container id is required");
+        }
+
+        if (!UUIDUtil.isUUID(id)) {
```

Comment:
This validation actually should not be here because then file containers would fail. 
This is an example of a call with file containers:
```PUT /api/v1/containers/_archive?containerId=/application/containers/test```

Reply:
Code has been updated by removing uuid validation and its curl/postman test